map_gui/src/colors.rs

```diff
@@ -191,7 +190,6 @@ impl ColorScheme {
             current_object: Color::WHITE,
             perma_selected_object: Color::BLUE,
             bottom_bar_id: Color::RED,
```

Comment:
Maybe once we can change fonts mid-line (https://github.com/a-b-street/abstreet/blob/ac615e97c3d25064ab0f9b7a335e64bdd40d02b8/widgetry/src/text.rs#L304 -- probably not too hard, and we can maybe even make usvg do more of the work with alignment), we can also get rid of the red debug IDs and make them bold (or flip the bold/underline IDs/names).

The bottom bar has never appeared in any of the Figma designs. :)  I'm solely responsible for all wacky colors there.